evap/evaluation/models.py

```diff
@@ -1552,6 +1552,14 @@ def full_name_with_additional_info(self):
             return name + " (" + self.email.split("@")[0] + ")"
         return name + " (" + self.email + ")"
 
+    @property
+    def sorting_key(self): # keep in sync with Meta#ordering
+        return (
+            (self.last_name or "").lower(),
+            (self.first_name or "").lower(),
+            (self.email or "").lower()
+        )
+
```

Comment:
I'd assume that when ordering by string, we usually want case insensitivity (@janno42?). In this case, [django supports case insensitive model ordering](https://code.djangoproject.com/ticket/6498#comment:24) using the [`Lower` function](https://docs.djangoproject.com/en/3.2/ref/models/database-functions/#lower).

For the three changed sort calls, we could then just use database sorting, which we get anyway. In staff/views.py, this would even gives us a chance for optimization, it currently looks like we have O(N) queries there.

Any design change we do here should probably be done uniformly across the code base (probably another issue)